### ISSUE TITLE
how to output the predicted right image

### ISSUE THREAD

**Issue by option-greek** [ClubAI/MonoDepth-PyTorch#21]
Hi,
1) Is there a way to output the right side image being constructed from the predicted disparity during the self training step ?
2) Is there a way to 'sharpen' the disparity map being produced ?

Thanks,
OG

**Comment by ajinkyakhoche** [comment]
@option-greek 
For 1. line 126 in loss.py, 'right_est' variable is exactly what you're looking for. 

For 2. EDIT: In Godard's original implementation, they downscale smoothness loss by 2**i, where 'i' is the scale. ClubAI's have in fact implemented it correctly. 

so maybe you can reduce the smoothing loss weight?
